scripts/curate_wakehealth.py

```diff
@@ -0,0 +1,159 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset msseg_challenge_2021 to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    args = parser.parse_args()
+    return args
+
+
+def create_json_sidecar(path_folder_sub_id_bids, item_out):
+    data_json = {"BodyPartDetails": "Tibialis nerves",
+                 "SampleStaining": "Toluidine Blue",
+                 "PixelSize": [0.226, 0.22753],
+                 "Environment": "exvivo",
+                 "Manufacturer": "Hamamatsu",
+                 "ManufacturersModelName": "NanoZoomer Slide Scanner",
+                 }
+    item_out = item_out.replace('_photo.tif', '_BF.json')
+    with open(os.path.join(path_folder_sub_id_bids, item_out), 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+
+def main(root_data):
+    output_data = os.path.join(root_data + '_curated')
+
+    # Remove macOS .DS_Store
+    os.system("find " + root_data + " -name '.DS_Store' -type f -delete")
+
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data)
+
+    subdatasets = os.listdir(root_data)
+    counter_move = 0
+    counter_file_in = 0
+
+    # Loop across subdirectories
+    for ds in subdatasets:
+        path_dataset = os.path.join(root_data, ds)
+        contents_ds = os.listdir(path_dataset)
+        contents_ds.sort()
+        roi_no = 1
+        previous_sub_id_bids = ''
+
+        # Loop across contents of each subdirectory
+        for item in contents_ds:
+            sub_id_initial = item.split(' - ')[0]
```

Comment:
We would like to keep only the first 5 digits. For some files in `20210308_extracts`, there is no dash, ex:
`17-268 94144536 - 2017-09-27 22.44.10_x40_z0_extract20210526114558-11.tif`, the sub-id should be `17268`.

Reply:
okay sounds good. I will tweak the script.